content/docs/user-guide/external-dependencies.md

```diff
@@ -54,11 +54,14 @@ $ dvc run -n download_file
 
 ```dvc
 $ dvc run -n download_file
-          -d ssh://user@example.com:/home/shared/data.txt \
+          -d ssh://user@example.com/abs/path/to/data.txt \
           -o data.txt \
-          scp user@example.com:/home/shared/data.txt data.txt
+          scp user@example.com:/abs/path/to/data.txt data.txt
 ```
 
+> ⚠️ Please notice `/abs/path/to/data` is a path relative to the SFTP root
```

Comment:
it reads strange absolute path that is relative :)

Reply:
Agree. Will address.